docs/src/intro/usage.md

```diff
@@ -36,6 +36,7 @@ Currently, Miden VM can be executed with the following subcommands:
 * `verify` - this will verify a previously generated proof of execution for a given program.
 * `compile` - this will compile a Miden assembly program (i.e., build a program [MAST](../design/programs.md)) and outputs stats about the compilation process.
 * `analyze` - this will run a Miden assembly program against specific inputs and will output stats about its execution.
+* `repl` - this will initiate the Repl tool for Miden VM
```

Comment:
Let's change this to: "this will initiate the Miden REPL tool." and make "REPL tool" a link to the new section below.